### ISSUE TITLE
Bug - Nobody gets ownership of /

### ISSUE THREAD

**Issue by kris9854** [ansible-lockdown/RHEL9-CIS#23]
**Describe the Issue**
Bug: The nobody account gets ownership of  /. Following the rhel8-cis hardning it should be skipped when nobody's path is set.

**Expected Behavior**
The role should ensure that all users own their home directories. Though this shouldn't be for the nobody account.

**Additional Notes**
Before change the task looks like:

````yaml
- name: "6.2.10 | PATCH | Ensure users own their home directories"
  file:
      path: "{{ item.dir }}"
      owner: "{{ item.id }}"
      state: directory
  with_items: "{{ rhel9cis_passwd }}"
  loop_control:
      label: "{{ rhel9cis_passwd_label }}"
  when:
      - min_int_uid | int <= item.uid
      - rhel9cis_rule_6_2_10
  tags:
      - skip_ansible_lint  # settings found on 6_2_7
      - level1-server
      - level1-workstation
      - automated
      - patch
      - users
      - rule_6.2.10
````

**Possible Solution**:

````yaml
- name: "6.2.10 | PATCH | Ensure users own their home directories"
  file:
      path: "{{ item.dir }}"
      owner: "{{ item.id }}"
      state: directory
  with_items: "{{ rhel9cis_passwd }}"
  loop_control:
      label: "{{ rhel9cis_passwd_label }}"
  when:
      - min_int_uid | int <= item.uid
      - item.id != 'nobody' #added from rhel8 role to fix nobody own /
      - rhel9cis_rule_6_2_10
  tags:
      - skip_ansible_lint  # settings found on 6_2_7
      - level1-server
      - level1-workstation
      - automated
      - patch
      - users
      - rule_6.2.10
````

**Comment by uk-bolly** [comment]
hi @kris9854 

Thank you for raising the issue, feedback like this helps us to improve. I believe this is resolved under a PR that is outstanding. I will follow this up and try to get the merge carried out asap.

Thank you again for the feedback it is appreciated.

regards

uk-bolly

**Comment by uk-bolly** [comment]
Hi @kris9854 

Thank you again for your feedback. This appears to have been resolved in #18 ? Please let us know if this is still an issue?

many thanks

uk-bolly

**Comment by kris9854** [comment]
Hej @uk-bolly, just forgot to close it after PR. Sorry
